test/unit/test.speech_to_text.v1.js

```diff
@@ -223,6 +223,27 @@ describe('speech_to_text', function() {
     });
   });
 
+  describe('recognizeWebM()', function() {
+    const session_path = '/v1/sessions/foo/recognize';
+    const payload = {
```

Comment:
Since the `session_path` and `payload` are only getting used in one test, I'd recommend moving them into that test.